test_conformance/compiler/test_compiler_defines_for_extensions.cpp

```diff
@@ -49,8 +49,10 @@ const char *known_extensions[] = {
     "cl_khr_subgroup_shuffle",
     "cl_khr_subgroup_shuffle_relative",
     "cl_khr_subgroup_clustered_reduce",
+    "cl_khr_subgroup_rotate"
     "cl_khr_extended_bit_ops",
     "cl_khr_integer_dot_product",
+    "cl_khr_extended_async_copies",
```

Comment:
Likewise, cl_khr_extended_async_copies is in Section 34 so I guess it could come just after cl_khr_subgroup_named_barrier (Section 33).